qemu/tests/numa_cpu.py

```diff
@@ -0,0 +1,142 @@
+import logging
+
+from virttest import error_context
+from virttest.utils_misc import NumaInfo
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Assign cpu to numa node with "-numa cpu", check the numa info in monitor
+    and guest os match with the qemu cli
+    """
+
+    def convert_cpu_topology_to_index(socket, die, core, thread):
+        """
+        Convert the cpu topology to cpu index list
+        """
+        def _get_boundary(value, max_value, weight):
```

Comment:
What does 'weight' mean? A doc string would be better.

Reply:
> What does 'weight' mean? A doc string would be better.

I think it is a very common name for, every bit have it's weight.  Anyway, add doc string is acceptable for me too.